pyomo/contrib/mindtpy/nlp_solve.py

```diff
@@ -174,18 +188,14 @@ def handle_NLP_subproblem_optimal(fixed_nlp, solve_data, config):
     elif config.strategy == 'GBD':
         add_gbd_cut(solve_data, config)
```

Comment:
Does this function even exist nowadays?

Reply:
Nope. This function doesn't exit. I think the whole Generalized Benders Decomposition method hasn't been implemented.

Reply:
Maybe we should comment this out